index.html

```diff
@@ -1,110 +1,16 @@
 ---
 layout: default
 ---   
-<div class="greenhead">Where can I see the RFP?</div>
+
+{% for faq in site.data.faq %}
```

Comment:
At this point I'd like to roll these comments into an issue to be addressed at a later date. Once the RFP is closed on May 8th, we can start moving towards a generalizable site.